.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
**Suggetion**

In _**lint.js**_ file, the indent rule is off according comment;

> Indentation is disabled pending a switch from 4 to 2 space for JS.

With that in mind, we can use .editorconfig file to instruct to use of 2 spaces.

Reply:
Hi! This change is unrelated to the rest of the PR/bug, so it is best to split this out to another PR and bug.